packages/zoe/package.json

```diff
@@ -61,6 +61,7 @@
   },
   "files": [
     "src/",
+    "contractFacet.js",
```

Comment:
Hmm, this is already in src/

Reply:
There's one in the package root, too.  That's for external clients to use `@agoric/zoe/contractFacet`.

Reply:
Oh, I see, thanks!